tvb/datatypes/projections.py

```diff
@@ -78,7 +124,17 @@ class ProjectionRegionEEG(framework.ProjectionRegionEEGFramework,
         
     
     """
-    pass
+    @staticmethod
+    def from_file(source_file='projection_EEG_region.npy', instance=None):
```

Comment:
ah I was mainly curious why not read directly from the .mat in bst db, but I agree where only one array is required. Leave it in npy then.